### ISSUE TITLE
Warm Blitz Cache Failed

### ISSUE THREAD

**Issue by Osze** [putyourlightson/craft-blitz#37]
Warm Blitz Cache fails on me every time. 

Blitz Version: 1.6.6 

Included URI Patterns:
/.*  (I want to cache every page)

Amount of pages: 304

I've tried this so far:
- Increased php memory from 256mb to 1024mb. 
- increased max_execution_time from 30 to 60
- Installed Async 
- Cleared storage/runtime

I'm running a m4.large server on Amazon with ServerPilot.

Any ideas what to try next?

Otherwise Blitz is amazing! :)

**Comment by Osze** [comment]
This is my htaccess. 

```
# 1 Month for most static assets
<filesMatch ".(jpg|jpeg|png|gif|js|ico|svg|woff|woff2)$">
    Header set Cache-Control "max-age=2592000, public"
</filesMatch>

<IfModule mod_rewrite.c>
    RewriteEngine On

   #Now, rewrite to HTTPS:
    RewriteCond %{HTTP:X-Forwarded-Proto} !=https
    RewriteRule .* https://%{HTTP_HOST}%{REQUEST_URI} [R=301,L]

    # Remove trailing slashes
    RewriteCond %{REQUEST_FILENAME} !-d
    RewriteRule ^(.*)/$ /$1 [L,R=301]

    #Redirect from www to non-www
    RewriteCond %{HTTP_HOST} ^www\.mydomain\.com [NC]
    RewriteRule (.*) https://mydomain.com/$1 [R=301,L]

    # Blitz cache rewrite
    RewriteCond %{DOCUMENT_ROOT}/cache/blitz/%{HTTP_HOST}/%{REQUEST_URI}/%{QUERY_STRING}/index.html -s
    RewriteCond %{REQUEST_METHOD} GET
    RewriteRule .* /cache/blitz/%{HTTP_HOST}/%{REQUEST_URI}/%{QUERY_STRING}/index.html [L]

    ## Send would-be 404 requests to Craft.
    RewriteCond %{REQUEST_FILENAME} !-f
    RewriteCond %{REQUEST_FILENAME} !-d
    RewriteCond %{REQUEST_URI} !^/(favicon\.ico|apple-touch-icon.*\.png)$ [NC]
    RewriteRule (.+) index.php?p=$1 [QSA,L]
</IfModule>
```

**Comment by putyourlightson** [comment]
In what way is it failing? Is the job reporting a "Failed" status? If so then can you check the error either in the response value in network tab of your browser console or in the `queue` table in the database. This plugin should also help with troubleshooting failed jobs:
https://github.com/lukeyouell/craft-queue-manager

If the issue is something else then please send more details or if you can send login credentials to info@putyourlightson.net then I can take a closer look.

**Comment by Osze** [comment]
Thanks. Will install Queue Manager and get back to you with more info.

**Comment by Osze** [comment]
Figured it out with Queue Manager:

First I got this error message:
The process "/usr/local/bin/php /app/bin/craft queue/exec "2137" "300" "1" "" --color=" exceeded the timeout of 300 seconds.

Change TTR to 600 by adding:
`'components' => [
        'queue' => [
            'ttr' => 600,
        ],
    ],`

to config/app.php

but what I think the real problem was all my entries that sent a 501 or 404 error. I've used some entries to point to other entries "Link To Entry" - type. When error pages was excluded it worked great!

Thanks for an awesome product!
